dpsim-models/src/SP/SP_Ph1_Resistor.cpp

```diff
@@ -107,44 +107,9 @@ void SP::Ph1::Resistor::mnaCompApplySystemMatrixStamp(
     SparseMatrixRow &systemMatrix) {
   Complex conductance = Complex(1. / **mResistance, 0);
 
-  for (UInt freq = 0; freq < mNumFreqs; freq++) {
-    // Set diagonal entries
-    if (terminalNotGrounded(0))
-      Math::addToMatrixElement(systemMatrix, matrixNodeIndex(0),
-                               matrixNodeIndex(0), conductance, mNumFreqs,
-                               freq);
-    if (terminalNotGrounded(1))
-      // Set off diagonal entries
-      Math::addToMatrixElement(systemMatrix, matrixNodeIndex(1),
-                               matrixNodeIndex(1), conductance, mNumFreqs,
-                               freq);
-    if (terminalNotGrounded(0) && terminalNotGrounded(1)) {
-      Math::addToMatrixElement(systemMatrix, matrixNodeIndex(0),
-                               matrixNodeIndex(1), -conductance, mNumFreqs,
-                               freq);
-      Math::addToMatrixElement(systemMatrix, matrixNodeIndex(1),
-                               matrixNodeIndex(0), -conductance, mNumFreqs,
-                               freq);
-    }
-
-    SPDLOG_LOGGER_INFO(mSLog, "-- Stamp frequency {:d} ---", freq);
-    if (terminalNotGrounded(0))
-      SPDLOG_LOGGER_INFO(mSLog, "Add {:s} to system at ({:d},{:d})",
-                         Logger::complexToString(conductance),
-                         matrixNodeIndex(0), matrixNodeIndex(0));
-    if (terminalNotGrounded(1))
-      SPDLOG_LOGGER_INFO(mSLog, "Add {:s} to system at ({:d},{:d})",
-                         Logger::complexToString(conductance),
-                         matrixNodeIndex(1), matrixNodeIndex(1));
-    if (terminalNotGrounded(0) && terminalNotGrounded(1)) {
-      SPDLOG_LOGGER_INFO(mSLog, "Add {:s} to system at ({:d},{:d})",
-                         Logger::complexToString(-conductance),
-                         matrixNodeIndex(0), matrixNodeIndex(1));
-      SPDLOG_LOGGER_INFO(mSLog, "Add {:s} to system at ({:d},{:d})",
-                         Logger::complexToString(-conductance),
-                         matrixNodeIndex(1), matrixNodeIndex(0));
-    }
-  }
+  MNAStampUtils::stampAdmittance(conductance, systemMatrix, matrixNodeIndex(0),
```

Comment:
I assumed here that iterating through mNumFreqs was a copy-paste error, because it is not done for other SP components.
@martinmoraga, @m-mirz

Reply:
It could be... I don't remember testing the multi frequency approach for SP.